src/test/store/receive-profile.test.js

```diff
@@ -82,6 +81,22 @@ function simulateSymbolStoreHasNoCache() {
   }));
 }
 
+/**
+ * Advance timers and promises used for receive-profile's _wait().
+ *
+ * On a profile fetch failure, we wait one second and try again. To
+ * facilitate testing, this code advances through these waits the
+ * given number of times. This is unfortunately highly coupled to the
+ * implementation details of receive-profile.
+ */
+async function runThroughWaitDelay(times: number) {
+  for (let i = 0; i < times; i++) {
+    await Promise.resolve();
+    await Promise.resolve();
+    jest.runAllTimers();
+  }
+}
```

Comment:
Replacing the `setTimeout` functionality was trickier than I thought, and this is the best way I could solve the problem.

Unfortunately I have no idea why two promise resolves are necessary. I read about it and I debugged, but I couldn't find out. So I'll leave it here as is, and hopefully someone else has an idea why, or a suggestion of a better way to do this.

Some helpful resources I had while troubleshooting:
- https://stackoverflow.com/questions/52177631/jest-timer-and-promise-dont-work-well-settimeout-and-async-function
- https://github.com/facebook/jest/issues/9181